java/registry/src/main/resources/application.yml.sample

```diff
@@ -157,7 +148,7 @@ signature:
     #file: ${audit_frame_file:audit_frame.json}
 
 authentication:
-  enabled: ${authentication_enabled:true}
+  enabled: ${authentication_enabled:false}
```

Comment:
is this value need to be put into system ENV or properties file.?

Reply:
Not a must, but can be overridden with env var.